engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -445,6 +451,8 @@ def __init__(self, **kwargs):
         self.waiting_steps = kwargs.get("waiting_steps", None)
         self.current_step = 0
         self.default_output_formatter = kwargs.get("output_formatter", None)
+        global TGI_COMPAT
+        TGI_COMPAT = kwargs.get("tgi_compat", "false").lower() == 'true'
```

Comment:
```suggestion
        self.tgi_compat = kwargs.get("tgi_compat", "false").lower() == 'true'
```

Reply:
There is no way to pass this value back to the serializer. Unless we make them to be part of the Request class